amr-wind/equation_systems/icns/icns_advection.H

```diff
@@ -576,6 +563,24 @@ struct AdvectionOp<ICNS, fvm::MOL>
             for (amrex::MFIter mfi(dof_field(lev), mfi_info); mfi.isValid();
                  ++mfi) {
                 amrex::Box const& bx = mfi.tilebox();
+                amrex::Box gbx = grow(bx, fvm::MOL::nghost_state);
+
+                // Set up momentum array
+                amrex::FArrayBox qfab;
```

Comment:
can we do: `amrex::FArrayBox qfab(gbx, ICNS:ndim);` instead of splitting on 2 lines?

Reply:
good idea. I'll make this change.